frontend/src/core/locale/getPluralExamples.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+
+import type { Locale } from './actions';
+
+
+export default function getPluralExamples(locale: Locale) {
```

Comment:
One more thing, it took me a long time to understand what this function returns, i.e., `{"1": 1}`, right? Can you add a comment or maybe even add an indexer type? https://flow.org/en/docs/types/objects/#toc-objects-as-maps